scripts/crowbar-testbuild.py

```diff
@@ -38,8 +38,14 @@
 }
 
 JOB_PARAMETERS = {
+    'crowbar-ha': ('nodenumber=4', 'hacloud=1',
```

Comment:
can you create variables for the the parameters used in ```crowbar-ha``` and ```barclamp-pacemaker``` (which are identical). Same for ```crowbar-ceph``` and ```barclamp-ceph```.